cli/src/cluster_query.rs

```diff
@@ -1876,9 +1902,13 @@ pub fn process_show_stakes(
             }
         }
     }
-    Ok(config
-        .output_format
-        .formatted_string(&CliStakeVec::new(stake_accounts)))
+    if stake_accounts.is_empty() {
+        Ok("No stake account found for provided vote account(s).".into())
```

Comment:
What if the user hasn't provided any vote accounts? (eg being filtered on a withdraw_authority that cannot be found)

Reply:
Alright, changed the message